src/model/generate_model.jl

```diff
@@ -88,32 +88,34 @@ function generate_model(setup::Dict,inputs::Dict,OPTIMIZER::MOI.OptimizerWithAtt
 
 	# Initialize Power Balance Expression
 	# Expression for "baseline" power balance constraint
-	@expression(EP, ePowerBalance[t=1:T, z=1:Z], 0)
+	createemptyexpression!(EP, :ePowerBalance, T, Z)
 
 	# Initialize Objective Function Expression
-	@expression(EP, eObj, 0)
+	temp = AffExpr(0.0)
+	EP[:eObj] = temp
```

Comment:
Can this be done as a single line?

Reply:
Yes, the similar code in `createemptyexpression!` uses a temporary variable to ensure type-stability but that isn't necessary here.